src/index.html

```diff
@@ -11,7 +11,130 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="fields">
+        <legend>Personal information</legend>
+        <div class="personal_info">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            id="surname"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="personal_info">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            id="name"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="personal_info">
+          <label for="number">How old are You?</label>
+          <input
+            type="number"
+            id="number"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </div>
+
+        <div class="personal_info">
+          <label for="date">Full date of birth:</label>
+          <input type="date" id="date">
+        </div>
+
+        <div class="personal_info">
+          <label for="term">I accept the term of the agreement</label>
+          <input type="checkbox" id="term">
+        </div>
+      </fieldset>
+
+      <fieldset class="fields">
+        <legend>Registration</legend>
+        <div class="personal_info">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div class="personal_info">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            id="password"
+            minlength="6"
+            maxlength="32"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="fields">
+        <legend>An interesting fact about you!</legend>
+        <div class="personal_info">
+          Do you love cats?
+          <label>
+            <input type="radio" name="cats"> Yes
+          </label>
+
+          <label>
+            <input type="radio" name="cats"> No
+          </label>
+        </div>
+
+        <div class="personal_info">
+          <label for="color">What is your favorite color?</label>
```

Comment:
I would stick to formatting your code like that:
```suggestion
          <label for="color">
            What is your favorite color?
          </label>
```

It is much more easy to read and, therefore, easier to change sth later if needed